src/views/workflow-page/config/workflow-page-tabs-contents-map.config.ts

```diff
@@ -1,6 +1,8 @@
 import type { WorkflowPageTabsContentsMap } from '../workflow-page-tab-content/workflow-page-tab-content.types';
 
-export const worflowPageTabsContentsMapConfig = {
+const worflowPageTabsContentsMapConfig = {
```

Comment:
```suggestion
const workflowPageTabsContentsMapConfig = {
```

Not sure how I didn't catch this before, but you can change it in this PR